features/docs/cli/fail_fast.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Fail fast
+
+  The --fail-fast flag causes Cucumber to exit immediately after the first 
+  scenario fails.
+
+  Scenario: When a scenario fails
```

Comment:
Yeah totally agreed about the use of that global state @brasmusson - it's a legacy we're living with for now but I'd love to see a PR that gets rid of it. Should be much easier now that the behaviour is all centred in the `QuitFilter`.